func/task11.js

```diff
@@ -0,0 +1,25 @@
+const square = (n) => (n * n);
+console.log(`Simple invoke square ${square(4)}`);
+
+const memorize = (func) => {
+    let cache = {};  
+
+    let result = function(...args) {
+        const arg = args[0];
```

Comment:
what if I use an object as an argument?
memorizeSquare({l: 4});
memorizeSquare({l: 4}).
will it work?